notebooks/data-sources/gcsweb-ci/build-logs/build_log_EDA.ipynb

```diff
@@ -0,0 +1,1093 @@
+{
```

Comment:
<p>Is this the raw output shown above? For example <code>Running step e2e-***-gather-audit-logs. </code>  Looks like it may have gone through Drain already. Is that the case or is this what some of the logs actually look like?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>It is (almost) raw output.</p><p> </p><p>Besides the 3 lines at the top that are printed to show the build ID and URL, the only other difference that you see is that the colours come from the escape sequences in the raw log:</p><p> </p><pre class="ql-syntax" spellcheck="false">^[[36mINFO^[[0m[2021-04-10T10:38:08Z] ci-operator version v20210410-6a59d22
^[[36mINFO^[[0m[2021-04-10T10:38:08Z] Loading configuration from ...

</pre><p>I am not sure how the names of the steps are "simplified" there. My guess is that the <code>ci-operator</code>  makes them simpler so that they match the step names in the <a href="https://docs.ci.openshift.org/docs/architecture/step-registry/" target="_blank">step registry</a>. The code that creates that output is <a href="https://github.com/openshift/ci-tools/blob/717fd9f8502d1a83905e9ac33f0e34501133c4fa/pkg/steps/multi_stage.go#L442" target="_blank">here</a> I believe, but honestly I have not looked at that part much... I was more worried about the diverse content that appears on failed builds.</p>

Reply:
I now added direct links to each raw log file when showing log excerpts.

Reply:
That is ok. I was just double checking to make sure some data that had been processed through Drain hadn't sneaked in there some how.